AraEvent/AraEventCalibrator.cxx

```diff
@@ -1838,7 +1840,7 @@ void AraEventCalibrator::loadAtriCalib(AraStationId_t stationId, Double_t unixti
 
     char epsilonFileName[100];
     sprintf(epsilonFileName,"%s/ATRI/araAtriStation%iEpsilon.txt",calibDir, stationId);
-    // fprintf(stdout, "AraEventCalibrator::loadAtriCalib(): INFO - Epsilon file = %s\n", epsilonFileName);//DEBUG
+    fprintf(stdout, "AraEventCalibrator::loadAtriCalib(): INFO - Epsilon file = %s\n", epsilonFileName);//DEBUG // uncommenting this -- Mohammad
```

Comment:
Do we want this to print everytime?

Reply:
Yes, I think that's a good debug check! We print pedestal, timing, voltage, and high-voltage paths. Epsilon files are also used in calibration. So, it's better we print that too alongside.

Reply:
Okay sounds good!